.env.example

```diff
@@ -0,0 +1,9 @@
+# Firebase Configuration
+FIREBASE_CREDENTIAL_FILE=./config/firebase-credentials-dev.json
+
+# Azure Storage Configuration
+AZURE_CONNECTION_STRING=devconnectionstring
```

Comment:
Is dit niet `UseDevelopmentStorage=true` ofzo?

Reply:
In principe zou dat ook moeten werken, maar bij mij niet haha.
Ik moet de default dev connection string gebruiken die exact hetzelfde doet en geen speciale"/secret waardes heeft (want het is voor iedereen dezelfde account key).
Dus ik zal het naar dit veranderen: "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;QueueEndpoint=http://127.0.0.1:10001/devstoreaccount1;TableEndpoint=http://127.0.0.1:10002/devstoreaccount1;"